### PR TITLE
feat: add download buttons

### LINKED ISSUE #15: Add download button
Replace to download bullet list with a download button which automatically links to your architecture. It should still be possible to download a version for a different architecture (e.g. with a drop-down arrow).

### PR DESCRIPTION
**Pull request by Polostor**
# Short description

New design for downloads added to the bottom of the website as requested in #15.

Closes #15.

# Changes

feat: add download buttons #15

# Static visuals

image

### CODE REVIEW

**Comment by Polostor** [comment]
Would like to hear your thought @Indy2222

**Comment by Polostor** [comment]
Added icons to make download buttons more obvious
image

Downside is that low quality (50px/52px pngs) and [licence](https://icons8.com/license) of using those images for free..

File: sass/styles.scss [hunk 1]
```diff
@@ -57,6 +63,60 @@ div.middle {
...
+            box-sizing: border-box;
+            flex-basis: 45%;
+            text-align: center;
+            background-color: #701e13;
+            border-radius: 10px;
+            border: 1px solid #ddb991;
+
+            &#download-steps {
+                background-color: transparent;
+                //flex-basis: 80%;
```
Comment (Indy2222) [inline sass/styles.scss]: Why is this commented out? Can we remove it?

File: sass/styles.scss [hunk 2]
```diff
@@ -57,6 +63,60 @@ div.middle {
...
+            margin: 16px;
+            padding: 16px 0;
+            box-sizing: border-box;
+            flex-basis: 45%;
+            text-align: center;
+            background-color: #701e13;
+            border-radius: 10px;
+            border: 1px solid #ddb991;
+
+            &#download-steps {
```
Comment (Indy2222) [inline sass/styles.scss]: Why does the steps have to be treated specially?

File: templates/index.html [hunk 3]
```diff
@@ -21,6 +21,24 @@ <h1>_coming&ZeroWidthSpace;_soon</h1>
...
+
+    <div><img src="icons8-linux-52.png" class="icon" /> <a href="https://download.de-game.org/x86_64-unknown-linux-gnu/nightly.zip">Linux<br/>(<code>x86_64-unknown-linux-gnu</code>)</a></div>
+    <div><img src="icons8-windows-10-50.png" class="icon" /><a href="https://download.de-game.org/x86_64-pc-windows-gnu/nightly.zip">Windows<br/>(<code>x86_64-pc-windows-gnu</code>)</a></div>
+    <div><img src="icons8-mac-50.png" class="icon" /><a href="https://download.de-game.org/aarch64-apple-darwin/nightly.zip">macOS with M series<br/>(<code>aarch64-apple-darwin</code>)</a></div>
+    <div><img src="icons8-mac-50.png" class="icon" /><a href="https://download.de-game.org/x86_64-apple-darwin/nightly.zip">macOS with Intel<br/>(<code>x86_64-apple-darwin</code>)</a></div>
+
+    <div id="download-steps">
+      <h3>Download steps</h3>
+      <ol>
+        <li>Download the ZIP file for your OS and CPU:</li>
```
Comment (Indy2222) [inline templates/index.html]: Please replace the colon with a period.

File: templates/index.html [hunk 4]
```diff
@@ -21,6 +21,24 @@ <h1>_coming&ZeroWidthSpace;_soon</h1>
...
+        <li>Extract the ZIP file.</li>
+        <li>Execute binary file called <code>de</code> or <code>de.exe</code>.</li>
+      </ol>
+    </div>
+  </div>
+
   <ol>
```
Comment (Indy2222) [inline templates/index.html]: Please remove this part as it is superseded with the above.

**Review by Indy2222 — CHANGES_REQUESTED** [review]

File: sass/styles.scss (hunk 1 above)
Comment (Polostor) [reply sass/styles.scss]: done

File: sass/styles.scss (hunk 2 above)
Comment (Polostor) [reply sass/styles.scss]: it's a special component to make it more obvious what to do next with download.

File: templates/index.html (hunk 3 above)
Comment (Polostor) [reply templates/index.html]: done

File: templates/index.html (hunk 4 above)
Comment (Polostor) [reply templates/index.html]: done

**Comment by Polostor** [comment]
Also the new changes improve download view on mobile devices

**Comment by Polostor** [comment]
One more thing - I bought the svgs for OS (windows, linux and apple) from https://icons8.com/icons/pricing .

I declare that hereby I give them to this project for usage on website (at this moment https://de-game.org ).

**Review by Indy2222 — APPROVED** [review]

**Pull request merged**